example/follows-mouse.md

```diff
@@ -0,0 +1,24 @@
+[width]: 400
+[height]: 400
+
+# Tooltip Follows Mouse
+
+In this example, the tooltip follows the mouse. It demonstrates the behavior of the tooltip at different positions on the screen, especially near the edges.
```

Comment:
add a line to the effect of "Move your mouse around the black box to test out the behavior!"